playbooks/roles/backups/files/backup.py

```diff
@@ -128,7 +128,7 @@ def compress_backup(backup_path):
     return compressed_backup_path
 
 
-def dump_service(service_name, backup_dir):
+def dump_service(service_name, backup_dir, user='', password=''):
```

Comment:
@johnbaldwin These will be empty if no username and password are specified on the command line or by environment variables. Empty strings implies that no authentication is required.